src/index.js

```diff
@@ -138,9 +138,17 @@ const handleCloudTrailLogs = (event, context, lintEnv) => {
 
 const handler = (event, context) => {
   const apiToken = process.env.LogIntelligence_API_Token;
+  const ingestionUrl = process.env.LogIntelligence_API_Url || 'https://data.cloud.symphony-dev.com/le-mans/v1/streams/ingestion-pipeline-stream';
+
+  if (!apiToken || !ingestionUrl) {
+    context.fail(error);
```

Comment:
I don't see the variable 'error' defined. I suggest we change the code to:

if (!apiToken || !ingestionUrl) {
  handleError('Neither API token nor ingestion URL can be missing. Please configure them in the lambda function');
}

Reply:
Corrected to follow same pattern as other errors. Left return value so it wouldn't continue execution